8. Appels réseaux/Content/part1.md

```diff
@@ -125,10 +125,10 @@ jsonp=<string> — callback function name, used for jsonp format only (usage exa
 
 Voyons ceux qui nous intéressent :
 - `method` : il a l'air obligatoire et c'est celui qui nous permet de récupération la citation avec la valeur `getQuote`.
-- `format` : le type de format que l'on souhaite pour la réponse. La documentation précise les différents formats disponibles. Je vous propose qu'on choisisse `json` car c'est un format de donnée très populaire dans le développement mobile pour sa petite taille.
+- `format` : le type de format que l'on souhaite pour la réponse. La documentation précise les différents formats disponibles. Je vous propose qu'on choisisse `json` car c'est un format de donnée très populaire dans le développement mobile.
```

Comment:
I would not say that JSON is popular because it's small, it's popular because it's simple. And you have options smaller than JSON, such as MessagePack